src/main/java/world/anhgelus/molehunt/Game.java

```diff
@@ -116,6 +132,14 @@ public void stop() {
     public void end() {
         timer.cancel();
         timer = new Timer();
+
+        borderTimeOffsetTime.cancel();
+        borderTimeOffsetTime = new Timer();

```

Comment:
If `borderTimeOffsetTask` is a TimerTask, it's not necessary to create a new one

Reply:
after cancelling a timer you can use it again ?

Reply:
You don't cancel the timer, you cancel the task

Reply:
ah